tbtadm/controller.cpp

```diff
@@ -74,9 +74,31 @@ const std::string opt_remove      = "remove";
 const std::string opt_remove_all  = "remove-all";
 const std::string opt_once_flag   = "--once";
 
-const std::string indent     = "|   ";
+const std::string indent     = "│   ";
+const size_t indentLength    = 4;
```

Comment:
`= indent.size()`?

Reply:
That does not work because .size() returns bytes not chars and I think the pipe char used above is unicode. I don't know if C++ strings have some method to return chars instead. Any ideas?

Reply:
Good point. Unfortunately, unicode handling is still complicated. Keeping this `4` makes sense then.